### ISSUE TITLE
Identificação do usuário não cadastrado

### ISSUE THREAD

**Issue by sahrosa** [servicosgovbr/editor-de-servicos#407]

**Comment by sahrosa** [comment]
Pessoal,
segue o mock up de como será essa página:

![screen shot 2015-11-18 at 4 24 42 pm](https://cloud.githubusercontent.com/assets/2932644/11250603/59ed4116-8e14-11e5-87eb-beb31ebc9347.png)

**Comment by sahrosa** [comment]
Acham que vale o @glauberramos começar esta história mesmo que sem a funcionalidade @gustavo-freitas @pedrombl? Pensei nisso porque aí podemos linkar a página e já ter o css encaminhado, mas sem a funcionalidade de logar sem cadastro.

**Comment by gustavo-freitas** [comment]
@sahrosa claro! toca ficha @glauberramos.

**Comment by gustavo-freitas** [comment]
Uma novo papel deve ser criado para esse usuário não cadastrado. Não haverá criação de tokens do lado do banco de dados. Todas as permissões devem seguir o cartão https://github.com/servicosgovbr/editor-de-servicos/issues/324. 

Os seguintes detalhes são bastante importantes:

- Gerenciar Permissões e Nova Página não devem exibidos
- Lista de Páginas deverá mostrar somente serviços
- Publicar/Despublicar/Descartar não devem ser exibidos. Somente Visualizar e Salvar devem ser exibidos
- Botão de Logout deve ser exibido (com o nome do cidadão)
- Como esse tipo de acesso não dá permissão para criar serviços, renomear deve apresentar mensagem de permissão inválida
- Permissões não devem ser associadas a um órgão específico
- O e-mail utilizado pelo cidadão na hora do login, deve ser o mesmo que irá aparecer no commit do Github

**Comment by sahrosa** [comment]
Um único ponto: será que não podemos esconder o botão de renomear serviço @gustavo-freitas?

**Comment by gustavo-freitas** [comment]
@sahrosa Sim! Entretanto, considerando que estamos focando em funcionalidade, não vejo isso como algo essencial. Podemos abrir um cartão para essa questão do botão de renomear. O que acha?

**Comment by sahrosa** [comment]
Combinado! Cartão #487 criado para tratar essa questão :)

**Comment by maiteb** [comment]
Para registro: os trabalhos nessa tarefa estão sendo realizados na branch ___usuario_nao_logado__.
Até o momento (24/12), temos a rotina de acesso de usuário não logado, faltando apenas duas coisas:
- A tela de acesso;
- Checar quais informações do usuário devem aparecer no commit do GitHub (nome e/ou email);

Assim que estes pontos forem endereçados, farei o merge da branch na master para começarmos os testes.

**Comment by maiteb** [comment]
Fiz um teste local, e as informações do usuário não logado (nome e email), estão aparecendo no commit do GitHub

**Comment by gustavo-freitas** [comment]
@andreleite @maiteb antes de passar para o Fabrício, precisamos ainda implementar a validação do CPF na hora que o usuário não cadastrado realiza login. CPF inválidos não devem poder realizar login.

**Comment by maiteb** [comment]
@gustavo-freitas vou tocar isso enquanto o Docker Hub está fora do ar

**Comment by maiteb** [comment]
@gustavo-freitas já enviei as alterações necessárias para funcionar a validação de CPF. 
@andreleite, você poderia dar uma olhadinha para mim no css da tela de Acesso Cidadão? Acho que falta só uns pequenos ajustes para ficar no padrão do erro da tela de login de usuário cadastrado

**Comment by gustavo-freitas** [comment]
Alterações realizadas no CSS.

**Comment by gustavo-freitas** [comment]
@almeidafab funcionalidade pronta! Alteração indo para Alpha. Ainda esse ano, faremos o deploy manual para DEV.

**Comment by josselmo** [comment]
Funcionou. OK
